python/metatensor-torch/metatensor/torch/__init__.py

```diff
@@ -18,6 +18,8 @@
     load = torch.ops.metatensor.load
     save = torch.ops.metatensor.save
 
+# PR COMMENT might need to include import of
+#            LabelsEntry also into the opeartions.py file
```

Comment:
not sure what you mean?

Reply:
Should I add `LabelsEntry` to `operations.py` to be consistent with `learn.py` even though we dont use `LabelsEntry` in metatensor-operations? I think otherwise you get a weird error if you do `from metatensor.torch.operatons import *` because it tries to import `LabelsEntry` since it is in the `__all__`. I think the answer would be yes

Reply:
Why is `LabelsEntry` in `__all__` for operations? It should come from either `metatensor` or `metatensor.torch`, not `metatensor.operations` or `metatensor.torch.operations`.

I agree that one should be able to do `from metatensor.torch.operations import *` (even if that's stupid) without error, but there might be another way to fix it.